src/Auth/BaseAuthenticate.php

```diff
@@ -37,6 +37,7 @@
      * - `userModel` The alias for users table, defaults to Users.
      * - `scope` Additional conditions to use when looking up and authenticating users,
      *    i.e. `['Users.is_active' => 1].`
+     * - `finder` The finder method to use to fetch user record. Defaults to 'all'.
```

Comment:
I agree that deprecating the scope and contain parameters makes sense.
On 3 Aug 2015 7:53 pm, "Mark Story" <notifications@github.com> wrote:

> In src/Auth/BaseAuthenticate.php
> <https://github.com/cakephp/cakephp/pull/7164#discussion_r36077773>:
>
> > @@ -37,6 +37,7 @@
> >       * - `userModel` The alias for users table, defaults to Users.
> >       * - `scope` Additional conditions to use when looking up and authenticating users,
> >       *    i.e. `['Users.is_active' => 1].`
> > +     * - `finder` The finder method to use to fetch user record. Defaults to 'all'.
>
> We could probably deprecate the contain & scope options. The new finder
> option gives people more control/flexibility than those other options.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cakephp/cakephp/pull/7164/files#r36077773>.
>